docs/index.md

```diff
@@ -0,0 +1,26 @@
+# GCM Documentation Index
+
+The following are links to GCM user support documentation:
+
+- [Frequently asked questions][gcm-faq]
+- [Command-line usage][gcm-usage]
+- [Configuration options][gcm-config]
+- [Environment variables][gcm-env]
+- [Enterprise configuration][gcm-enterprise-config]
+- [Network and HTTP configuration][gcm-net-config]
+- [Credential stores][gcm-credstores]
+- [Host provider specification][gcm-host-provider]
+- [Azure Repos OAuth tokens][gcm-azure-tokens]
+- [GitLab support][gcm-gitlab]
+
+[gcm-azure-tokens]: docs/azrepos-users-and-tokens.md
+[gcm-config]: docs/configuration.md
+[gcm-credstores]: docs/credstores.md
+[gcm-dev]: docs/development.md
+[gcm-enterprise-config]: docs/enterprise-config.md
+[gcm-env]: docs/environment.md
+[gcm-faq]: docs/faq.md
+[gcm-gitlab]: docs/gitlab.md
+[gcm-host-provider]: docs/hostprovider.md
+[gcm-net-config]: docs/netconfig.md
+[gcm-usage]: docs/usage.md
```

Comment:
Now these links originate from a file already in `/docs` we need to drop that prefix.

```suggestion
[gcm-azure-tokens]:azrepos-users-and-tokens.md
[gcm-config]: configuration.md
[gcm-credstores]: credstores.md
[gcm-dev]: development.md
[gcm-enterprise-config]: enterprise-config.md
[gcm-env]: environment.md
[gcm-faq]: faq.md
[gcm-gitlab]: gitlab.md
[gcm-host-provider]: hostprovider.md
[gcm-net-config]: netconfig.md
[gcm-usage]: usage.md
```

Reply:
Gar! Thank you.